.github/workflows/notify-pr.yml

```diff
@@ -0,0 +1,21 @@
+name: Pull Request Slack Notifier
+
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send notification to Slack
+        uses: slackapi/slack-github-action@v1.24.0
+        with:
+          payload: |
+            {
+              "action": "${{ github.event.action }}",
+              "url": "${{ github.event.pull_request.html_url || github.event.head_commit.url }}"
```

Comment:
Should we include title and status? Status to inform on the closed PR too.

Reply:
The action parameter includes `opened, reopened, synchronize`, and the title is included in the unfurling.